test/api/v3/integration/groups/POST-groups_groupId_quests_accept.test.js

```diff
@@ -0,0 +1,54 @@
+import {
+  createAndPopulateGroup,
+  translate as t,
+} from '../../../../helpers/api-v3-integration.helper';
+
+describe.skip('POST /groups/:groupId/quests/accept', () => {
```

Comment:
Ignore this file for now, it's just got some placeholder garbage until `invite` is in good shape.